compliance-by-design-demo/src/components/FactChangeModal.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+
+import Button from "react-bootstrap/Button";
+import FormControl from "react-bootstrap/FormControl";
+import InputGroup from "react-bootstrap/InputGroup";
+import Modal from "react-bootstrap/Modal";
+
+export default class FactChangeModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: "",
+    };
+  }
+
+  setValue(event) {
+    this.setState({
+      value: event.target.value,
+    });
+  }
+
+  handleSave() {
+    console.log("Saving", this.props.factName, this.state.value);
+    if (this.props.handleSave) {
+      if (typeof this.props.oldValue === "number") {
+        this.props.handleSave(this.props.factName, Number(this.state.value));
+      } else {
```

Comment:
Not yet implemented for strings?

Reply:
`this.state.value` is a string, so in the case of a string no conversion is necessary

Reply:
Yeah, but also nothing happens currently it seems

Reply:
Gotcha. I forgot to retest the string case after testing the number case. Good catch:)